docs/tutorial/installation.rst

```diff
@@ -38,6 +40,9 @@ http://localhost:15672/. The username is ``guest`` and the password is
 read the messages in the queues. Keep it open in a browser tab, we'll need it
 later.
 
+If your project uses containers, consult the `RabbitMQ documentation`_ on it's conatiner.
```

Comment:
There seem to be a typo on the last word. Also, it's probably a "its" ;-) but you may want to rephrase it as `consult the RabbitMQ documentation about containers`.